packages/core/src/payment/payment-strategy-registry.spec.ts

```diff
@@ -79,14 +80,8 @@ describe('PaymentStrategyRegistry', () => {
             expect(registry.getByMethod(getAmazonPay())).toBeInstanceOf(AmazonPayPaymentStrategy);
         });
 
-        it('returns credit card strategy if none is registered with method name', () => {
-            expect(registry.getByMethod(getBraintree())).toBeInstanceOf(CreditCardPaymentStrategy);
-        });
-
-        it('returns new strategy instance if multiple methods belong to same type', () => {
-            expect(registry.getByMethod(getBraintree())).toBeInstanceOf(CreditCardPaymentStrategy);
-            expect(registry.getByMethod(getBraintreePaypal())).toBeInstanceOf(CreditCardPaymentStrategy);
-            expect(registry.getByMethod(getBraintree())).not.toBe(registry.getByMethod(getBraintreePaypal()));
```

Comment:
Hi , @davidchin . I have:

- Fliped the order of V1 and V2 registrys, modified unit tests accordingly.
- Removed the old credit-card payment method from registry as it is no longer the default payment method.
- Tested all changes on a store.

https://user-images.githubusercontent.com/88361607/205209741-0c85feef-4559-4f63-b356-1a8bb3a117f9.mov

Just want to point out, because V1 won't return a default method now, this unit test has been moved. 
https://github.com/bigcommerce/checkout-sdk-js/blob/92bd1e66ec20d6cd4bbace5eb1af789ebd5c9b72/packages/core/src/common/registry/resolve-id-registry.spec.ts#L68.